.github/workflows/ci.yml

```
@@ -0,0 +1,56 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.18
+      uses: actions/setup-go@v3
+      with:
+        go-version: ^1.18
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v3  
+    - name: Build
+      run: |
+       make gomod
+       make build
+    - name: Debug
+      run: |
+        pwd
+        echo ${GOPATH}
+        echo ${GOROOT}
+      env:
+        GOPATH: /home/runner/work/cli/go
+  golangci:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

We  could upgrade to use v3 now (https://github.com/golangci/golangci-lint-action)

```suggestion
        uses: golangci/golangci-lint-action@v3
```

---

We need to setup go before if we upgrade to a newer version. Example:

```
- uses: actions/setup-go@v3
        with:
          go-version: 1.18
```